apcd-cms/src/client/src/components/Extensions/EditExtensionModal/EditExtensionModal.tsx

```diff
@@ -0,0 +1,380 @@
+                    />
+                    <small
+                      className="form-text text-muted"
+                      style={{ fontStyle: 'italic' }}
+                    >
+                      Current:{' '}
+                      {extension.approved_expiration_date
+                        ? new Date(
+                            extension.approved_expiration_date
+                          ).toLocaleDateString()
+                        : 'None'}
+                    </small>
+                    <ErrorMessage
+                      name="approved_expiration_date"
+                      component="div"
+                      className="text-danger"
+                    />
+                  </FormGroup>
+                </Col>
+                <Col md={6}>
+                  <FormGroup>
+                    <Label for="ext_status" className={styles.customLabel}>
+                      <strong>Extension Status</strong>
+                    </Label>
+                    <Field
+                      as="select"
+                      name="ext_status"
+                      id="ext_status"
+                      onChange={formik.handleChange}
+                      onBlur={formik.handleBlur}
+                      className={`form-control ${styles.viewRecord}`}
+                      value={formik.values.ext_status}
+                    >
+                      {statusOptions?.map(
+                        (opt) =>
+                          opt.value !== 'All' && (
+                            <option
+                              className="dropdown-text"
+                              key={opt.key}
+                              value={opt.value}
+                            >
+                              {opt.value}
+                            </option>
+                          )
+                      )}
+                    </Field>
+                    <ErrorMessage
+                      name="ext_status"
+                      component="div"
+                      className="text-danger"
+                    />
+                  </FormGroup>
+                </Col>
+                <Col md={6}>
+                  <FormGroup>
+                    <Label for="ext_outcome" className={styles.customLabel}>
+                      <strong>Extension Outcome</strong>
+                    </Label>
+                    <Field
+                      as="select"
+                      name="ext_outcome"
+                      id="ext_outcome"
+                      onChange={formik.handleChange}
+                      onBlur={formik.handleBlur}
+                      className={`form-control ${styles.viewRecord}`}
+                      value={formik.values.ext_outcome}
+                    >
+                      {outcomeOptions?.map((opt) => (
+                        <option
+                          className="dropdown-text"
+                          key={opt.key}
+                          value={opt.value}
```

Comment:
I'm getting an error here that opt is a string and does not have a key or value.